sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -340,11 +359,6 @@ export interface EntityRecognitionSkill {
 // @public
 export type EntityRecognitionSkillLanguage = 'ar' | 'cs' | 'zh-Hans' | 'zh-Hant' | 'da' | 'nl' | 'en' | 'fi' | 'fr' | 'de' | 'el' | 'hu' | 'it' | 'ja' | 'ko' | 'no' | 'pl' | 'pt-PT' | 'pt-BR' | 'ru' | 'es' | 'sv' | 'tr';
 
-// @public
-export interface ETagOperationOptions {
-    accessCondition?: AccessCondition;
```

Comment:
wouldn't it be a simpler change to just modify this interface to be `onlyIfUnchanged?: boolean` to avoid repeating the typing on every interface?

Reply:
Yes. I just wanted to keep it consistent with other SDKs and did not want to wrap it up one level under